adventure.py

```diff
@@ -11,7 +12,7 @@
 dim = int(input("Tile dimension?\n")) # getting world dimensions from user
 name = input("Please enter your name:  ")
 
-if platform.system() == "Windows":  # option to turn off colors to improve performance
+if platform.system() == "Windows"or platform.system() == "Linux":  # option to turn off colors to improve performance
```

Comment:
Are colors automatically turned off for OS X, and have you tested to make sure that this will work on Linux in the first place? If you try and import the module and linux doesn't support it, you'll get an error.

Reply:
Yes. Most linux python installs pre-include colorama, and are therein compatible with our local usage of it. Tested on Bob's RasPi.

Reply:
okay.